src/Raygun.Tests.Blazor.Server/appsettings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "Raygun": {
+        "ApiKey": "APIKEY",
```

Comment:
`ApiKey` is set on purpose on this test project, even if the connection is mocked, the client still checks if the apikey is provided